### PR TITLE
Homework6: Aleksandr Strokov

### PR DESCRIPTION
**Pull request by alstrokov**

### CODE REVIEW

File: src/task01.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+from persons import Lector
+from persons import Student
+
+
+def main():
+    lector = Lector("Max", 33)
+    stu1 = Student("Nick", 20)
+    stu2 = Student("Ada", 19)
+    group = Group("Q19")
+    group.members.append(stu1)
```
Comment (belovmd) [inline src/task01.py]: usually we use methods to add them (add_member, exclude_member i.e.)

File: src/materials.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+    def prepare(self, topic):
+        self.topic = topic
+
+
+class Hometask(Material):
+    tasks = []
+
+    def __init__(self, name):
+        super().__init__(name)
```
Comment (belovmd) [inline src/materials.py]: we don't need super here as parent super would be called if we don't defint init in this class

File: src/materials.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    @abstractmethod
+    def prepare(self, *param):
+        pass
+
+
+class Lecture(Material):
+    topic = ''
+
+    def __init__(self, name):
+        super().__init__(name)
```
Comment (belovmd) [inline src/materials.py]: redundant init

File: src/persons.py [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+        self.age = age
+
+    @abstractmethod
+    def greet(self):
+        pass
+
+
+class Student(Person):
+    def __init__(self, name, age):
+        super().__init__(name, age)
```
Comment (belovmd) [inline src/persons.py]: redundant super

**Review by belovmd — COMMENTED** [review]
please add docstrings to classes and methods
please address these issues

**Comment by alstrokov** [comment]
ready to review

**Review by belovmd — COMMENTED** [review]
8,5 (maximum score for this task)

**Pull request closed without merging**